Automattic/pocket-casts-android
# Bookmarks - Display for user episodes

## Description

This displays bookmarks for user episodes (files).

There's no Figma design, but here's the iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/1018

## Testing Instructions
- Open a user episode (files)
- Play it and add a few bookmarks
- Return to the Files screen
- ✅ Notice that the file has a bookmark icon
- Tap on the file row
- ✅ Notice that the file details bottom sheet has a bookmark menu
- Tap on the bookmarks menu
- ✅ Notice that the bookmarks dialog is shown with bookmarks for the selected file

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/8084a6a6-7017-4593-b83f-654b589037ed

#### Known Issues:
- Theming doesn't work well
- On deleting all bookmarks for a file, the bookmark icon is not removed for it until the view is reloaded
- Reuses sort order previously set for episodes 

## Checklist
 
#### I have tested any UI changes...
